docs/userguide/format_parse/messages/try-mf2.md

```diff
@@ -0,0 +1,366 @@
+
+These instructions will use a release version of ICU4C (tested with 76.1) and
+a "Hello ICU world" project already created.
+They provide minimal effort that only requires downloading and opening in Visual Studio
+before using.
+
+> :point_right: **Note**: the MessageFormat 2 implementation in a previously release version may be behind the spec in the first few releases of ICU after the MF2.0 spec was declared 1.0 in CLDR 46.1,
+which occurred between ICU 76 and ICU 77. The difference between latest MF2.0 and the spec version supported in ICU may be minimal.
+
+
+1. Download the Visual Studio artifacts from the
+[official release of ICU 76.1](https://github.com/unicode-org/icu/releases/tag/release-76-1):
+    * [icu4c-76_1-Win32-MSVC2022.zip](https://github.com/unicode-org/icu/releases/download/release-76-1/icu4c-76_1-Win32-MSVC2022.zip)
+    * [icu4c-76_1-Win64-MSVC2022.zip](https://github.com/unicode-org/icu/releases/download/release-76-1/icu4c-76_1-Win64-MSVC2022.zip)
+
+1. Download the "Hello ICU / MF2 World" project [HelloMF2.zip](HelloMF2.zip).
+
+1. Unzip the files you just downloaded and merge the content of the two ICU folders.
+The "hello world" project to be a sibling to the icu folder.
+
+1. The folder tree structure should look like this
+    
+    ```
+    someFolderOfYourChoice\
+      +- HelloMF2\
+      | \- HelloMF2\
+      \- icu\
+        \- icu4c\
+          +- bin\
+          +- bin64\
+          +- include\
+          | \- unicode\
+          +- lib\
+          \- lib64\
+    ```
+
+1. Open the `HelloMF2.sln` solution in Visual Studio and you are ready to go.
+
+
+## Java
+
+We will assume that if you are interested in testing a pre-release Java library
+you already have (or know how to install) a JDK, Apache Maven, git,
+know how to create a project in your favorite IDE, and so on.
+
+### What you need
+
+* JDK, any version between 8 and 23
+* Apache Maven
+
+### Instructions
+
+1. Create a new Maven project
+    ```sh
+    mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=hello_icu_mf2 -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.5 -DinteractiveMode=false
+
+    cd hello_icu_mf2
+    ```
+
+1. Edit the `pom.xml` file
+
+    1. The project created as above uses the Java 17 version.
+    If you are using a lower version then change `<maven.compiler.release>` property to whatever Java version you are using.
+        ```sh
+        # Example on how to set the Java version to 11
+        mvn versions:set-property -Dproperty=maven.compiler.release -DnewVersion=11
+        ```
+
+    1. Add this to `<dependencies>`
+        ```xml
+        <dependency>
+          <groupId>com.ibm.icu</groupId>
+          <artifactId>icu4j</artifactId>
+          <version>77.0.1-SNAPSHOT</version>
+        </dependency>
+        ```
+        **Warning:** make sure it is done in `dependencies`, not in `dependencyManagement / dependencies`
+
+1. Change the `src/test/java/com/mycompany/app/AppTest.java` file
```

Comment:
Nitpick: "Edit" instead of change?

It does not matter, but this is the same thing we do to the pom.xml when we add the dependencies.
If we edit there, we edit here.
Or we "change" in both places.

Reply:
Done.